src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -94,6 +94,7 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
                 var configuration = new SessionConfiguration(
                     circularBufferSizeMB: buffersize,
                     format: EventPipeSerializationFormat.NetTrace,
+                    requestRundown: true,
```

Comment:
What about adding an option `--no-rundown`, by default `false`?

Reply:
I initially thought about doing this, except if you pass `true` to that, for now you can't open the file in PerfView (I'm guessing it's because PerfView expects certain rundown events to be there). Until further investigation I don't want to expose that and cause a broken end-to-end experience :-)

Reply:
I did not know `PerfView` would fail to open the file.